web/gui/dashboard_info.js

```diff
@@ -3408,6 +3410,124 @@ netdataDashboard.context = {
         'The ratios (in %) are tracked as recent trends over 10-, 60-, and 300-second windows.'
     },
 
+    // ------------------------------------------------------------------------
+    // containers (systemd)
+
+    'services.cpu': {
```

Comment:
This morning @vlvkobal required me to change my PR that is also working with `systemd`/`cgroups` with the goal to have descriptions not only for `systemd` but also for `containers`, as you can see [here](https://github.com/netdata/netdata/pull/11611#discussion_r721238414). I suggest we use charts context here to avoid the same problem with these descriptions.

Reply:
That is the chart context.